### PR TITLE
update ch20

### PR DESCRIPTION
**Pull request by nmambre**

### CODE REVIEW

**Comment by jonthegeek** [comment]
Don't merge this quite yet if you happen to see it, @lgibson7 and @Ken-Vu . Nelson is going to push one more commit in about an hour (after our meeting).

**Review by lgibson7 — APPROVED** [review]
Looks good. Thanks @nmambre

**Pull request merged**
